go-controller/pkg/util/util_unit_test.go

```diff
@@ -10,6 +10,8 @@ import (
 
 	ovntest "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/testing"
 
+	"regexp"
```

Comment:
can you move this import into the first block?

Reply:
done